evalml/model_understanding/graphs.py

```diff
@@ -652,6 +652,12 @@ def partial_dependence(
         X_features = (
             X.ww.iloc[:, [features]] if isinstance(features, int) else X.ww[[features]]
         )
+    X_unknown = X_features.ww.select("unknown")
+    if len(X_unknown.columns):
+        # We drop the unknown columns in the pipelines, so we cannot calculate partial dependence for these
+        raise ValueError(
+            "Columns included for partial dependence cannot be of type 'Unknown'"
```

Comment:
I think one helpful tidbit here would probably be to tell the user in the exception which columns are Unknown